source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -1,41 +1,39 @@
 .. _web-server-install:
 
-Creating a Clear Linux based web server
+Create a Clear Linux based web server
 #######################################
 
-Prerequisites
-*************
+This tutorial guides you through creating a LAMP server as part of setting up a Clear Linux-based web server. It details how to install and configure each LAMP server component and then create an initial WordPress MySQL database.
```

Comment:
I like the idea of adding a brief intro for context. 
I recommend: 
"This tutorial shows you how to create a LAMP server on |CL| and how to use phpMyAdmin to manage an associated database. "

[Otherwise, the current wording could be potentially misleading.]

Reply:
Since we're messing with the web tutorial, I'd also like to add the alternative of using nginx since that is the goto website engine these days instead of apache.  Be careful what you ask for! :)